src/api.ts

```diff
@@ -44,6 +44,7 @@ export default class Chromeless<T extends any> implements Promise<T> {
         closeTab: true,
         ...options.cdp,
       },
+      chromeFlags: ['--headless'],
```

Comment:
what when someone does not want `--headless` mode, then it's not possible to override flags? Anyway I am open to that change because I will never want to run chrome without headless flag :)